src/application/deployer/push_application_images.go

```diff
@@ -0,0 +1,52 @@
+package deployer
+
+import (
+	"path"
+
+	"github.com/Originate/exosphere/src/aws"
+	"github.com/Originate/exosphere/src/docker/tools"
+	"github.com/Originate/exosphere/src/types"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ecr"
+)
+
+// PushApplicationImagesOptions is the options to PushApplicationImages
+type PushApplicationImagesOptions struct {
```

Comment:
It feels a little more clear to me (but not that much better) if we call these files and their respective structs `PushImage`/`push_images.go` and `PushImage`/`push_image.go`. 

Also, this struct seems a little unnecessary as it only contains two fields that are only used once by `PushApplicationImages`. It would almost make more sense to me to pass in two params instead of overloading the words `options` and having slightly confusing struct names.

What do you think?

Reply:
I'll move the options struct into its own file. The options is only used for what goes into the public function.

I'm good passing in it as two arguments